interface/client/lib/helpers/helperFunctions.js

```diff
@@ -114,7 +114,7 @@ Helpers.generateBreadcrumb = function (url) {
         return el === '';
     });
 
-    return new Spacebars.SafeString(_.flatten(['<span>' + filteredUrl.host + ' </span>', pathname]).join(' ▸ '));
+    return new Spacebars.SafeString(url.protocol +'//'+ _.flatten(['<span>' + filteredUrl.host + ' </span>', pathname]).join(' ▸ '));
```

Comment:
Will do, but as far as i understand whats put into the protocol is already stripped out. if invalid, it would be returned empty by webkit.

But we can certainly do it to increase security